vera.galstyan/repo/Lesson_3/exercise_08/exercise_08.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+int(input("enter tree numbers"))
```

Comment:
CSW: The message should start with uppercase and have a semicolon (:)  and space after semicolon